packages/dev/cypress/e2e/unovis.cy.ts

```diff
@@ -0,0 +1,358 @@
+describe('Unovis Test', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('Load homepage', () => {
+    cy.percySnapshot('Homepage test')
+  })
+
+  it('Basic Annotation', () => {
+    cy.contains('Basic Annotations').click()
+    cy.wait(3000)
+    cy.percySnapshot('Basic Annotation', { scope: '.exampleViewer' })
```

Comment:
I would move `{ scope: '.exampleViewer' }` to a dedicated variables since it is repeated so many times